pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/DropFieldStepTest.java

```diff
@@ -258,54 +258,6 @@ void testKeyValueBytesJson() throws Exception {
         "{\"valueField3\":\"value3\"}");
   }
 
-  @Test
-  void testAvroNotModified() throws Exception {
```

Comment:
Yes. Because now  when AVRO is used, we always convert to NATIVE AVRO, even if the key/value was not modified.
Since we will serialize to bytes later, it doesn't affect performance.
But I can check if the previous behaviour can be kept if needed.